tasks/celery.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- name: Ensure celeryd pid default directory is present
+  file:
+    state: directory
+    owner: "{{ django_system_user }}"
+    group: "{{ django_system_group }}"
+    path: "{{ item }}"
+  when:
+    - django_celeryd_pid_file is none
+  with_items:
+    - "/var/run/celery-{{ django_system_user }}"
```

Comment:
Can we make this configurable or use the directory under `django_celeryd_pid_file` or https://github.com/onaio/ansible-django/blob/8edb9c0aa5865aecf4f6f2f3eebf712160be0ab3/templates/etc/default/celeryd.j2#L25 means that the default directory is used if `CELERYD_PID_FILE` is not set?

Reply:
Yes, and the default is going to be `/var/run/celery-{{ django_system_user }}`

Reply:
From Celery docs, the `CELERYD_PID_FILE`
> Default is /var/run/celery/%n.pid

Reply:
I don't see us create the django_celeryd_pid_file` directory too